m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.RPT.BATCH

```diff
@@ -154,6 +337,12 @@ SETUP
 *]
  TESTACCOUNTS=""
 
+[* DEBUGMODE: If set to TRUE and TESTACCOUNTS are being utilized, prints out
+** additional info pretaining to Preference Access records and transaction
```

Comment:
@tkainz at line 341 typo in word "pretaining" in sentence "... additional info pretaining to Preference Access records..."

Reply:
@jordan-keenan Corrected in the code waiting to be committed - pending peer from Robby.